SWLOR.Game.Server/Service/CraftService/RecipeType.cs

```diff
@@ -815,7 +815,7 @@ public enum RecipeType
         CookedBibikibo = 2091,
         SlicedSardine = 2092,
         FishBroth = 2093,
-        SardineBall = 2094,
```

Comment:
Need a migration for this change. Anyone who already crafted the Sardine Ball will break after updating this.